### PR TITLE
Chromium 66

### PR DESCRIPTION
**Pull request by alexeykuzmin**
Project board: https://github.com/electron/electron/projects/10
Electron PR: https://github.com/electron/electron/pull/12477

### CODE REVIEW

File: script/create-dist [hunk 1]
```diff
@@ -130,73 +130,28 @@ BINARIES_SHARED_LIBRARY = {
...
-    os.path.join('obj', 'third_party', 'webrtc', 'rtc_base', 'rtc_base_approved_generic_cc.pdb'),
-    os.path.join('obj', 'third_party', 'webrtc', 'rtc_base', 'rtc_base_generic_cc.pdb'),
     os.path.join('obj', 'third_party', 'webrtc', 'rtc_base', 'rtc_base_generic.lib'),
-    os.path.join('obj', 'third_party', 'webrtc_overrides', 'init_webrtc_cc.pdb'),
     os.path.join('obj', 'third_party', 'webrtc_overrides', 'init_webrtc.lib'),
-    os.path.join('obj', 'third_party', 'zlib', 'zlib_c.pdb'),
-    os.path.join('obj', 'third_party', 'zlib', 'zlib_adler32_simd_c.pdb'),
-    os.path.join('obj', 'third_party', 'zlib', 'zlib_x86_simd_c.pdb'),
-    os.path.join('obj', 'ui', 'events', 'dom_keycode_converter_cc.pdb'),
-    os.path.join('obj', 'ui', 'events', 'dom_keycode_converter.lib'),
```
Comment (alexeykuzmin) [inline script/create-dist]: @gavignus I guess we still need `.lib` files )

**Comment by sebakerckhof** [comment]
Chrome 66 has [new policies](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes) around autoplaying of audio, which don't really make sense for electron applications. Will these be disabled or will there be a way to disable them?

**Comment by alexeykuzmin** [comment]
@sebakerckhof Can you please create a new issue in the `electron/electron` repo with the same question? Thanks!
https://github.com/electron/electron/issues/new

**Comment by CapOM** [comment]
Any chance an early beta tag can be made for testing purpose ? I mean one that can be build with current electron master, thx!

**Pull request merged**